src/test/java/keycontacts/storage/JsonAdaptedStudentTest.java

```diff
@@ -119,21 +123,21 @@ public void toModelType_invalidPianoPieces_throwsIllegalValueException() {
         List<JsonAdaptedPianoPiece> invalidPianoPieces = new ArrayList<>(VALID_PIANO_PIECES);
         invalidPianoPieces.add(new JsonAdaptedPianoPiece(INVALID_PIANO_PIECE));
         JsonAdaptedStudent student = new JsonAdaptedStudent(VALID_NAME, VALID_PHONE, VALID_ADDRESS, VALID_GRADE_LEVEL,
-                invalidPianoPieces, VALID_REGULAR_LESSON);
+                invalidPianoPieces, VALID_REGULAR_LESSON, VALID_CANCELLED_LESSONS);
         assertThrows(IllegalValueException.class, student::toModelType);
     }
 
     @Test
     public void toModelType_emptyRegularLesson_success() throws Exception {
         JsonAdaptedStudent student = new JsonAdaptedStudent(VALID_NAME, VALID_PHONE, VALID_ADDRESS, VALID_GRADE_LEVEL,
-                VALID_PIANO_PIECES, EMPTY_REGULAR_LESSON);
+                VALID_PIANO_PIECES, EMPTY_REGULAR_LESSON, VALID_CANCELLED_LESSONS);
         assertEquals(BENSON.withRegularLesson(null), student.toModelType());
     }
 
     @Test
     public void toModelType_invalidRegularLesson_throwsIllegalValueException() {
         JsonAdaptedStudent student = new JsonAdaptedStudent(VALID_NAME, VALID_PHONE, null, VALID_GRADE_LEVEL,
-                VALID_PIANO_PIECES, INVALID_REGULAR_LESSON);
+                VALID_PIANO_PIECES, INVALID_REGULAR_LESSON, VALID_CANCELLED_LESSONS);
         assertThrows(IllegalValueException.class, student::toModelType);
     }
```

Comment:
I see that you didn't add a test here. I realise that it is not very useful since you have it tested in JsonAdaptedCancelledLesson. So I'm just going to leave this comment here, if you want to add it in just add it, if not it should be fine